variables.tf

```
@@ -316,3 +316,22 @@ variable "storage_accounts" {
     error_message = "Storage account type must be either \"AzureFiles\" or \"AzureBlob\"."
   }
 }
+
+variable "public_network_access_enabled" {
+    description = "Should public access be enabled for this Web App?"
+    default = true
+    type = bool
+}
```

- Update order of arguments
- Update description
- Update formatting

```suggestion
variable "public_network_access_enabled" {
    description = "Should public network access be enabled for this Web App?"
    type        = bool
    default     = true
}
```